.cirrus.yml

```diff
@@ -77,7 +77,12 @@ ws_scan_task:
   depends_on:
     - build
   gke_container:
-    <<: *CONTAINER_DEFINITION
+    dockerfile: .cirrus/python-env.Dockerfile
+    builder_image_project: language-team
+    builder_image_name: docker-builder-lt-v1
+    cluster_name: cirrus-ci-lt-cluster
+    zone: us-central1-a
+    namespace: default
```

Comment:
Since this is the same container definition as for the build task, it may be worth storing it in something like BUILD_CONTAINER_DEFINITION to be more maintainable.

Reply:
Indeed, makes sense! thanks :)